cf/query.py

```diff
@@ -315,6 +345,8 @@ def __dask_tokenize__(self):
         operator = self._operator
         if operator == "isclose":
             value += (self.rtol, self.atol)
+        if operator == "wi":
```

Comment:
Just a bit of house style - we generally have a  blank line after clauses:
```suggestion

        if operator == "wi":
```